src/test/java/picard/sam/SamErrorMetric/CollectSamErrorMetricsTest.java

```diff
@@ -422,28 +446,10 @@ public void testOneCovariateErrorMetrics(final String errorSubscript, final File
         };
     }
 
-    @Test(dataProvider = "OneCovariateIndelErrorMetricsDataProvider")
+    @Test(dataProvider = "oneCovariateIndelErrorMetricsDataProvider")
     public void testOneCovariateIndelErrorMetrics(final String errorSubscript, final File samFile, final int priorQ, BaseErrorMetric expectedMetric) {
-        final File referenceFile = CHR_M_REFERENCE;
-        final File vcf = new File(TEST_DIR, "NIST.selected.vcf");
 
-        final File outputBaseFileName = new File(OUTPUT_DATA_PATH, "test");
-        final File errorByAll = new File(outputBaseFileName.getAbsolutePath() + errorSubscript);
-        errorByAll.deleteOnExit();
```

Comment:
It's hard to see in the diff but it looks like this file is no longer deleted on exit, is that correct?